server/src/main/java/com/paypal/selion/proxy/SeLionRemoteProxy.java

```diff
@@ -212,17 +216,6 @@ public void afterSession(TestSession session) {
                 appendMsgToCustomLog("Completed session #" + totalSessionsCompleted);
             }
             appendMsgToCustomLog("Total Number of slots used : " + getTotalUsed() + " on node :" + getId());
-            if (shutDownInProgress) {
-                appendMsgToCustomLog("Proceeding with shutdown of node " + getId());
-                if (getTotalUsed() <= 1) {
-                    shutdownNode();
-                    if (forceShutDown) {
```

Comment:
The admin shutdown (via /grid/admin/GridForceRestartDelegateServlet) path did not change with this PR and should be still be fully functional.   I have tested this just now on mac.

This is now merged.